ts/Series/ColumnPyramid/ColumnPyramidSeries.ts

```diff
@@ -228,9 +228,9 @@ class ColumnPyramidSeries extends ColumnSeries {
 
             // inverted chart
             if (chart.inverted) {
-                invBarPos = chart.plotWidth - barY;
+                invBarPos = yAxis.width - barY;
                 stackHeight = (topPointY -
-                (chart.plotWidth - (translatedThreshold as any)));
+                (yAxis.width - (translatedThreshold as any)));
```

Comment:
@TorsteinHonsi shouldn't the linter catch the lack of indentation here?

It would be more readable written this way:
```js
stackHeight =
    (topPointY - (yAxis.width - (translatedThreshold as any)));
```
or even this way:
```js
stackHeight = (
    topPointY - (yAxis.width - (translatedThreshold as any))
);
```